draft-ietf-quic-http.md

```diff
@@ -305,10 +304,55 @@ encouraged to coordinate their experiments on the quic@ietf.org mailing list.
 
 ## Discovering an HTTP/3 Endpoint {#discovery}
 
+HTTP relies on the notion of an authoritative response: a response that has been
+determined by (or at the direction of) the origin server identified within the
+target URI to be the most appropriate response for that request given the state
+of the target resource at the time of response message origination.  Locating an
+authoritative server for an HTTP URL is discussed in Section 5.4 of
+{{!SEMANTICS}}.
+
+The "https" scheme associates authority with the ability of a server to use a
+private key associated with a certificate that the client considers to be
+trustworthy for the host identified by the authority component of the URL. If a
+server presents a certificate that verifiably applies to the host, along with
+proof that it controls the corresponding private key, then a client will accept
+a secured connection to that server as being authoritative for all origins with
+the same scheme and host.
+
+When an "https" URI is used within a context that calls for access to the
+indicated resource, a client MAY attempt access by resolving the host identifier
+to an IP address, establishing a QUIC connection to that address on the
+indicated port, and sending an HTTP/3 request message to the server over that
+secured connection containing the URI's identifying data.
+
+Although HTTP is independent of the transport protocol, the "http" scheme
+associates authority with the ability to receive TCP connections on the
+indicated port of whatever host is identified within the authority component.
+Because HTTP/3 does not use TCP, HTTP/3 cannot be used for direct access to the
+authoritative server.  However, protocol extensions such as {{!ALTSVC=RFC7838}}
+permit the authoritative server to identify other services which are also
+authoritative and which might be reachable over HTTP/3.
+
+Prior to making requests for an origin whose scheme is not "https," the client
+MUST ensure the server is willing to serve that scheme.  If the client intends
+to make requests for an origin whose scheme is "http", this means that it MUST
+obtain a valid `http-opportunistic` response for the origin as described in
+{{!RFC8164}} prior to making any such requests.  Other schemes might define
+other mechanisms.
+
+Connectivity problems (e.g. firewall blocking UDP) can result in QUIC connection
+establishment failure; clients SHOULD attempt to use TCP-based versions of HTTP
+in this case.
+
+Servers MAY serve HTTP/3 on any UDP port; an alternative always includes
```

Comment:
```suggestion
Servers MAY serve HTTP/3 on any UDP port; an alternative service advertisement always includes
```

Not sure about the use of "advertisement" as that is not in the established nomenclature, but it's hard to distinguish between the service being provided and the declaration of that service being available.